uc-crux-llvm/test/Test.hs

```diff
@@ -0,0 +1,608 @@
+        ("memset_const_len_arg_byte.c", [("memset_const_len_arg_byte", isSafeWithPreconditions DidntHitBounds)]),
+        ("mutually_recursive_linked_list_sum.c", [("mutually_recursive_linked_list_sum", isSafeWithPreconditions DidHitBounds)]),
+        ("not_double_free.c", [("not_double_free", isSafeWithPreconditions DidntHitBounds)]),
+        ("ptr_as_array.c", [("ptr_as_array", isSafeWithPreconditions DidntHitBounds)]),
+        ("sized_array_arg.c", [("sized_array_arg", isSafeWithPreconditions DidntHitBounds)]),
+        ("writes_to_arg.c", [("writes_to_arg", isSafeWithPreconditions DidntHitBounds)]),
+        ("writes_to_arg_conditional.c", [("writes_to_arg_conditional", isSafeWithPreconditions DidntHitBounds)]),
+        ("writes_to_arg_conditional_ptr.c", [("writes_to_arg_conditional_ptr", isSafeWithPreconditions DidntHitBounds)]),
+        ("writes_to_arg_field.c", [("writes_to_arg_field", isSafeWithPreconditions DidntHitBounds)]),
+        ("writes_to_arg_ptr.c", [("writes_to_arg_ptr", isSafeWithPreconditions DidntHitBounds)]),
+        ("do_exit.c", [("do_exit", isUnclassified)]), -- goal: isSafe
+        ("do_fork.c", [("do_fork", isUnclassified)]),
+        ("do_getchar.c", [("do_getchar", isUnclassified)]), -- goal: isSafe
+        ("do_recv.c", [("do_recv", isUnclassified)]),
+        ("free_with_offset.c", [("free_with_offset", isUnclassified)]), -- goal: hasBugs
+        ("memset_arg_len.c", [("memset_arg_len", isUnclassified)]), -- goal: isSafeWP
+        ("nested_structs.c", [("nested_structs", isUnclassified)]), -- goal: ???
+        ("null_dereference.c", [("null_dereference", isUnclassified)]), -- goal: hasBugs
+        ("oob_read_heap.c", [("oob_read_heap", isUnclassified)]), -- goal: hasBugs
+        ("oob_read_stack.c", [("oob_read_stack", isUnclassified)]), -- goal: hasBugs
+        ("uninitialized_stack.c", [("uninitialized_stack", isUnclassified)]), -- goal: hasBugs
+        ("write_const_global.c", [("write_const_global", isUnclassified)]), -- goal: hasBugs
+        ("use_after_free.c", [("use_after_free", isUnclassified)]), -- goal: hasBugs
+        --
+        --
+        -- TODO(lb): Fix upstream? Missing annotations just seems like a bug.
+        ("compare_ptr_to_int.c", [("compare_ptr_to_int", hasMissingAnn)]), -- goal: hasBugs
+        ("compare_ptrs_different_heap_allocs.c", [("compare_ptrs_different_heap_allocs", hasMissingAnn)]), -- goal: hasBugs
+        ("compare_ptrs_different_stack_allocs.c", [("compare_ptrs_different_stack_allocs", hasMissingAnn)]), -- goal: hasBugs
+        ("memcpy_const_len.c", [("memcpy_const_len", hasMissingAnn)]),
+        ("deref_arg_arg_index.c", [("deref_arg_arg_index", hasMissingAnn)])
+        -- SQLite
+        -- This is slow, and WIP
+        -- inFile
+        --   "sqlite-3.32.1/sqlite3.c"
+        --   [ ("appendText", isSafeWithPreconditions False),
+        --     ("sqlite3_filename_database", isUnclassified)
+        --   ]
```

Comment:
Maybe we can remove the sqlite3.c files for now? That might tip the balance and turn Crucible from a Rust project into a C project. For the longer tests, could we potentially track them as submodules instead of including the source directly?

Reply:
Yeah I thought I would use them but haven't yet. Will remove.

Reply:
Removed, just need to remove this commented-out code.

Reply:
https://github.com/GaloisInc/crucible/pull/673/commits/520178b0b7112701881430c1edc3772d9037e84c